README.md

```diff
@@ -496,6 +496,18 @@ if it would make the expression longer or if the substitution parameters are
 anything but simple names or dotted names (as this can decrease readability).
 
 
+### Rewrite `xml.etree.cElementTree` to `xml.etree.ElementTree`
+
+Availability:
+- `--py36-plus` is passed on the commandline.
```

Comment:
this can be `--py3-plus` -- we have other older py3 target versions that are included there as well (mock, yield from, etc.)